packages/clerk-js/src/ui/components/SignUp/SignUpContinue.tsx

```diff
@@ -47,27 +47,27 @@ function _SignUpContinue() {
 
   // TODO: This form should be shared between SignUpStart and SignUpContinue
   const formState = {
-    firstName: useFormControl('firstName', '', {
+    firstName: useFormControl('firstName', initialValues?.firstName || '', {
```

Comment:
can we set a default of `{}` to initialValues in order to avoid the continuous optional chaining ?

Reply:
That wouldn't work, but we can set each property as `''`.

Reply:
I mean we could have this
```suggestion
    firstName: useFormControl('firstName', initialValues.firstName || '', {
```

Reply:
I made this change